volume-pulseaudio/volume-pulseaudio

```diff
@@ -22,17 +22,19 @@ LONG_FORMAT=${LONG_FORMAT:-'${SYMB} ${VOL}% [${INDEX}:${NAME}]'}
 SHORT_FORMAT=${SHORT_FORMAT:-'${SYMB} ${VOL}% [${INDEX}]'}
 USE_ALSA_NAME=${USE_ALSA_NAME:-0}
 USE_DESCRIPTION=${USE_DESCRIPTION:-0}
+JSON_FORMAT=${JSON_FORMAT:-0}
```

Comment:
i3blocks already has a `format` environmental variable builtin. Most blocks now use `format=pango` but in your case you  would put `format=json` in the i3blocks.conf file. Please use this `format` instead of creating a new one.

Reply:
If I understand the docs corectly, pango should go in `markup` not `format`. I've moved colour to pango.

Reply:
You are correct, pango is a markup thing not a format thing. However, from the man page of i3blocks:

     format This property specifies the format of the output text. The default format is plain text, as described in the  COMMAND  sec‐
                  tion. If "json" (or 1) is used, the block output is parsed as JSON.

The `format` key is a real key already part of i3blocks and `format=json` is the option that should be used in this case.